cpp/src/parquet/metadata_test.cc

```diff
@@ -66,7 +66,6 @@ std::unique_ptr<parquet::FileMetaData> GenerateTableMetaData(
   // column metadata
   col1_builder->SetStatistics(stats_int);
   col2_builder->SetStatistics(stats_float);
-  dict_encoding_stats.clear();
```

Comment:
Why remove this line? Since `/*has_dictionary=*/false` below, perhaps we need to pass empty dict encoding stats?